### PR TITLE
empty title means no title (logo only), so title header doesn't take space

### PR DESCRIPTION
**Pull request by doc-rj-celltrak**
When title is empty, this removes the empty space where title would normally be so that the logo spacing looks good.

### CODE REVIEW

File: example/lib/login_screen.dart [hunk 1]
```diff
@@ -40,10 +40,10 @@ class LoginScreen extends StatelessWidget {
     );
 
     return FlutterLogin(
-      title: Constants.appName,
+      title: '',
```
Comment (doc-rj-celltrak) [inline example/lib/login_screen.dart]: oops, i'll remove the changes in example

Comment (doc-rj-celltrak) [reply example/lib/login_screen.dart]: Removed now.

File: README.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ Property |   Type     | Desciption
 onSignup |   `AuthCallback`     | <sub>Called when the user hit the submit button when in sign up mode</sub>
 onLogin |   `AuthCallback`     | <sub>Called when the user hit the submit button when in login mode</sub>
 onRecoverPassword |   `RecoverCallback`     | <sub>Called when the user hit the submit button when in recover password mode</sub>
-title |   `String`     | <sub>The large text above the login [Card], usually the app or company name</sub>
+title |   `String`     | <sub>The large text above the login [Card], usually the app or company name, or empty string for no title</sub>
```
Comment (NearHuscarl) [inline README.md]: Replace the part you added with
```. Leave the string empty or null if you want no title```

File: lib/flutter_login.dart [hunk 3]
```diff
@@ -103,8 +103,7 @@ class _Header extends StatelessWidget {
     }
 
     Widget header;
-
-    if (titleTag != null) {
+    if (titleTag != null && title.isNotEmpty) {
```
Comment (NearHuscarl) [inline lib/flutter_login.dart]: check both empty and null string
```
if (titleTag != null && !DartHelper.isNullOrEmpty(title)) {
```

File: lib/flutter_login.dart [hunk 4]
```diff
@@ -113,11 +112,13 @@ class _Header extends StatelessWidget {
         style: theme.textTheme.display2,
         viewState: ViewState.enlarged,
       );
-    } else {
+    } else if (title.isNotEmpty) {
```
Comment (NearHuscarl) [inline lib/flutter_login.dart]: same as above
```
} else if (!DartHelper.isNullOrEmpty(title)) {
```

File: README.md (hunk 2 above)
Comment (doc-rj-celltrak) [reply README.md]: Done

File: lib/flutter_login.dart (hunk 3 above)
Comment (doc-rj-celltrak) [reply lib/flutter_login.dart]: Done

File: lib/flutter_login.dart (hunk 4 above)
Comment (doc-rj-celltrak) [reply lib/flutter_login.dart]: Done

**Pull request merged**